spec/ddtrace/contrib/rails/rack_spec.rb

```diff
@@ -137,13 +137,18 @@ def internal_server_error
       # We remove these during testing, as we are more interested in asserting
       # controller and template spans.
       super().reject { |s| SYNTHETIC_3_2_SPANS.include?(s.resource) }
+    # elsif true # TODO change this
+    #   super().reject { |s| SYNTHETIC_6_SPANS.include?(s.resource) }
```

Comment:
Was this intended to stay as TODO?

Reply:
Nope, I left commented out code, cleaning it up now.